hazelcast/src/test/java/com/hazelcast/config/yaml/W3cDomTest.java

```diff
@@ -47,11 +48,13 @@
  */
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
-public class W3cDomTest {
+public class W3cDomTest extends HazelcastTestSupport {
     private static final int NOT_EXISTING = 42;
 
     @Test
-    public void testYamlExtendedTestFromInputStream() {
+    public void testW3cDomAdapter() {
+        assumeThatJDK8();
```

Comment:
It won't run on JDK9+ I think, do we need sth like : 
```
assumeThatNoJDK6();
assumeThatNoJDK7();
```

Reply:
Good point. Shuffled the things a bit, renamed `assumeThatJdk8()` to `assumeThatJdk8OrHigher()` and changed it's logic accordingly, see the updated description.